topics/yamlconfig.rst

```diff
@@ -205,6 +205,73 @@ documentation <doxygen>`.
   describe the component. However, there is support for reflection by
   mnemonic name.
 
+  For example, you could have a stellar disk component and a dark matter halo component. 
+  The **dark halo** is the dark matter halo with a spherical basis, where parameters can 
+  be set as follows:
+
+.. code-block:: python
```

Comment:
``.. code-block:: yaml``  ??

I'm pretty sure that pygments has a yaml lexer...I believe I used it in `yamlconfig.rst`.